blocks/cards/cards.css

```diff
@@ -123,6 +123,10 @@
     display: flex;
   }
 
+  .card .card-thumbnail img {
```

Comment:
could you please prefix this with `.cards`? It will make sure it doesn't get applied to other places where someone might add a `card` CSS class.